content/200-orm/050-overview/500-databases/850-planetscale.mdx

```diff
@@ -139,9 +144,34 @@ One issue to be aware of is that [implicit many-to-many relations](/orm/prisma-s
 
 </Admonition>
 
+### Option 2: Enable foreign key constraints in the PlanetScale settings
+
+Support for foreign key constraints in PlanetScale databases has been Generally Available since February 2024. Follow the instructions in the [PlanetScale documentation](https://planetscale.com/docs/concepts/foreign-key-constraints) to enable them in your database.
+
+You can then use Prisma ORM and define relations in your Prisma schema without the need for extra configuration.
+
+In that case, you can define a relation as with other database that supports foreign key constraints, for example:
+
+```prisma file=schema.prisma
+model Post {
+  id       Int       @id @default(autoincrement())
+  title    String
+  content  String
+  likes    Int       @default(0)
+  comments Comment[]
+}
+
+model Comment {
+  id      Int    @id @default(autoincrement())
+  comment String
+  postId  Int
+  post    Post   @relation(fields: [postId], references: [id], onDelete: Cascade)
+}
+```
+
```

Comment:
Suggestion: Add another section here that repeats what does _not_ need to be done. No `relationMode`, no manual indexes, Introspection will pick up relations correctly.

Reply:
good point, I added some notes about this at the end of the section

<img width="418" alt="image" src="https://github.com/prisma/docs/assets/4058327/bd0d3a37-db65-40d1-ba34-2e3927de7f97">